src/constants/urls/api.js

```diff
@@ -1 +1,6 @@
 export const baseUrl = import.meta.env.VITE_BASE_URL;
+
+export const ENDPOINTS = {
```

Comment:
💊 제안
상수 네이밍시 단어를 구분해주시는 것을 추천드려요!
```suggestion
export const END_POINTS = {
```